artemis-journal/src/main/java/org/apache/activemq/artemis/core/io/aio/AIOSequentialFileFactory.java

```diff
@@ -304,7 +304,7 @@ public void run() {
          try {
             libaioFile.write(position, bytes, buffer, this);
          } catch (IOException e) {
-            callback.onError(-1, e.getMessage());
```

Comment:
hmmm.. nice catch...

I though other things would catch the failures..


The proper fix should been:

```
            callback.onError(-1, errorMessage);
            onIOError(e, "Failed to write to file", sequentialFile);
```